### PR TITLE
Clarify that HTTPS is not just needed for VoIP

### PR DESCRIPTION
**Pull request by rugk**
…but for basic security.

### CODE REVIEW

**Comment by MTRNord** [comment]
The pr should go against the develop branch.

File: README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ released version of Riot:
 Releases are signed by PGP, and can be checked against the public key
 at https://riot.im/packages/keys/riot.asc
 
-Note that Chrome does not allow microphone or webcam access for sites served
-over http (except localhost), so for working VoIP you will need to serve Riot
```
Comment (t3chguy) [inline README.md]: It'd be nice if the exception for localhost didn't disappear

File: README.md [hunk 2]
```diff
@@ -24,9 +24,11 @@ released version of Riot:
...
-over https.
+Note that for the security of your chats will need to serve Riot
+over HTTPS. Mayor browsers also do not allow you to use VoIP/video
+chats over HTTP, as WebRTC is only usable over HTTPS.
+An exception of this is localhost, which usually is alos trusted in
```
Comment (MTRNord) [inline README.md]: Typo "alos". Propably should mean "also" but I think it is not needed in the sentence at all

File: README.md [hunk 3]
```diff
@@ -24,9 +24,11 @@ released version of Riot:
 Releases are signed by PGP, and can be checked against the public key
 at https://riot.im/packages/keys/riot.asc
 
-Note that Chrome does not allow microphone or webcam access for sites served
-over http (except localhost), so for working VoIP you will need to serve Riot
-over https.
+Note that for the security of your chats will need to serve Riot
+over HTTPS. Mayor browsers also do not allow you to use VoIP/video
```
Comment (t3chguy) [inline README.md]: typo s/Mayor/Major/

**Comment by jonathancross** [comment]
Ping @rugk still have a comment not responded to.

File: README.md (hunk 1 above)
Comment (rugk) [reply README.md]: the last sentence actually mentions thus

**Comment by rugk** [comment]
Sorry, seems I've missed the notifications. :smile: 

Finally, improved/fixed.

**Comment by jryans** [comment]
@rugk If you have time to apply the suggestion above or mark the PR so that maintainers can edit and do so, then I believe we can press the merge button after that. 😁

**Comment by rugk** [comment]
Done. FYI "Allow edits from maintainers." had already been aliowed/selected.

**Comment by turt2live** [comment]
for some reason it was blocking me from doing both in and out of github :(

Thanks!

**Pull request merged**